suite-native/module-device-onboarding/src/components/DeviceOnboardingScreenWithExitButton.tsx

```diff
@@ -21,9 +21,15 @@ type NavigationProps = StackToStackCompositeNavigationProps<
     RootStackParamList
 >;
 
-const DeviceOnboardingExitButtonScreenHeader = () => {
+type DeviceOnboardingExitButtonScreenHeaderProps = {
+    onContinueButtonPress?: () => void;
```

Comment:
nit:
```suggestion
    onAlertContinueButtonPress?: () => void;
```
since the component is not containing only the `Alert`, more descriptive naming seems useful to me.

Reply:
27c740d902bfe3dc997f1e3b08838144de2b32db